bosh-stemcell/spec/support/os_image_shared_examples.rb

```diff
@@ -442,14 +442,6 @@
         expect(sshd_config.content).to match(/^PasswordAuthentication no$/)
       end
     end
-
-    context 'unless: softlayer', {
```

Comment:
@EdwardStudy looks like we deleted this test for everyone. we should bring it back since we do want to check that PermitRootLogin is set to no.

Reply:
@cppforlife I just saw same case in: https://github.com/bluebosh/bosh-linux-stemcell-builder/blob/6a40fe1aa2e00671c92052dae507d8b519a3ddec/bosh-stemcell/spec/support/os_image_shared_examples.rb#L206-L208. And I saw our pipeline passed case above. Did I miss something?